media_softlet/linux/common/ddi/media_libva_util_next.cpp

```diff
@@ -306,7 +306,17 @@ VAStatus MediaLibvaUtilNext::GenerateGmmParamsForNoneCompressionExternalSurface(
     MosUtilities::MosZeroMemory(&gmmCustomParams, sizeof(gmmCustomParams));
     gmmCustomParams.Type          = RESOURCE_2D;
     gmmCustomParams.Format        = ConvertMediaFmtToGmmFmt(params.format);
-    gmmCustomParams.BaseWidth64   = params.width;
+    if ((params.format == Media_Format_YV12) || \
+        (params.format == Media_Format_I420) || \
+        (params.format == Media_Format_IYUV) || \
+        (params.format == Media_Format_NV12) || \
+        (params.format == Media_Format_NV21)) {
```

Comment:
How is Media_Format_P010?

Reply:
suppose all 4:2:0 should be aligned, because the UV is half pixel.

Reply:
Added.

Reply:
If so, P012 and P016 require this too.
```
Media_Format_P012
Media_Format_P016
```

These formats should also be added in media_driver path, no?

https://github.com/intel/media-driver/blob/25a68cb146b6e5310c6b5e1bd202d5e9690635a9/media_driver/linux/common/ddi/media_libva_util.cpp#L524-L534